run_action.py

```diff
@@ -174,19 +174,40 @@ def main():
 
     github_token = os.environ.get("INPUT_GITHUB_TOKEN")
     github_api_url = os.environ.get("GITHUB_API_URL")
+    
     request_url = "%s/repos/%s/issues/%s/comments" % (
         github_api_url,
         repo,
         args.pull_request_id,
     )
 
+    headers = {
+        "Authorization": "token %s" % github_token,
+    }
+
+    one_comment = os.environ.get("INPUT_ONE_COMMENT")
+    if one_comment.lower() in ("true", "1"):
+        ## Search comments
+        requset_url_list_comments = requests.get(request_url,
+                                                headers=headers
+                                                ).json()
+
+        id_bot = 41898282
```

Comment:
What if you look for the ["header" of the comment](https://github.com/platisd/duplicate-code-detection-tool/blob/master/run_action.py#L167) and edit the first comment that you find containing `"## 📌 Duplicate code detection tool report\n"`?

It's rather unlikely that someone else has made a similar comment first and if they have, they are just messing with the Action so it's technically their fault.